orca-eureka/src/main/java/com/netflix/spinnaker/orca/eureka/DiscoveryStatusPoller.java

```diff
@@ -1,84 +0,0 @@
-/*
- * Copyright 2015 Netflix, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License")
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.netflix.spinnaker.orca.eureka;
-
-import java.util.Arrays;
-import java.util.List;
-import com.netflix.appinfo.InstanceInfo.InstanceStatus;
-import com.netflix.discovery.DiscoveryClient;
-import com.netflix.discovery.StatusChangeEvent;
-import com.netflix.spinnaker.kork.eureka.EurekaStatusChangedEvent;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.ApplicationListener;
-import org.springframework.context.event.ContextRefreshedEvent;
-import rx.Observable;
-import rx.Scheduler;
-import static com.netflix.appinfo.InstanceInfo.InstanceStatus.UNKNOWN;
-import static java.util.concurrent.TimeUnit.SECONDS;
-
-public class DiscoveryStatusPoller implements ApplicationListener<ContextRefreshedEvent> {
```

Comment:
We are getting reliable events from discovery client as to status changes (at least for application STARTING -> UP) but I need to test this a bit more with a proper eureka server and remote status toggle